### PR TITLE
Add ais response mappers for commerzbank, selection of account id corresponding to iban in postman

### PR DESCRIPTION
**Pull request by artemsnisarenko**

### CODE REVIEW

File: adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java [hunk 1]
```diff
@@ -55,7 +61,8 @@ private String getAccessToken() {
                                                                   RequestHeaders requestHeaders,
                                                                   RequestParams requestParams) {
         requestHeaders = withAuthorizationBearer(requestHeaders);
-        return super.getTransactionList(accountId, requestHeaders, requestParams);
+        return super.getTransactionList(accountId, requestHeaders, requestParams, CommerzbankTransactionsReport.class,
```
Comment (spetrychenko) [inline adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java]: super can be omitted

File: adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java [hunk 2]
```diff
@@ -69,6 +76,7 @@ private String getAccessToken() {
     @Override
     public GeneralResponse<BalanceReport> getBalances(String accountId, RequestHeaders requestHeaders) {
         requestHeaders = withAuthorizationBearer(requestHeaders);
-        return super.getBalances(accountId, requestHeaders);
+        return super.getBalances(accountId, requestHeaders, CommerzbankBalanceReport.class,
```
Comment (spetrychenko) [inline adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java]: super can be omitted

File: adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java (hunk 1 above)
Comment (artemsnisarenko) [reply adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java]: Removed

File: adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java (hunk 2 above)
Comment (artemsnisarenko) [reply adapters/commerzbank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/CommerzbankAccountInformationService.java]: Removed

**Pull request merged**